.github/workflows/ci.yml

```diff
@@ -47,6 +47,7 @@ jobs:
           _NORMALIZED_ENV_NAME=$(echo $DATA_PROJECT_DIR | rev | cut -d "/" -f 1 | rev | tr '.-' '_')
           GIT_BRANCH=${GITHUB_HEAD_REF}
           cat .tinyenv >> $GITHUB_ENV
+          echo >> $GITHUB_ENV # Ensures new line
```

Comment:
could we cat + echo at the end of this step?

I'm thinking there's no easy way now to override `_ENV_FLAGS` and other env variables, that way we could have `_ENV_FLAGS=` in `.tinyenv` and a user could disable data branching in some specific data project.

Reply:
Sure, I move it down to allow env vars overwrites. Therefore, new line shouldn't be required but I kept it to avoid problems in the future.

Reply:
Thanks!